File: official/docs/java/current/referral-customers/refund-by-payment-log.java
package referral;

import com.easypost.exception.EasyPostException;
import com.easypost.model.BetaPaymentRefund;
import com.easypost.service.EasyPostClient;

public class Create {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient(System.getenv("EASYPOST_API_KEY"));

        BetaPaymentRefund refund = client.refundByPaymentLog.refundByAmount(2000);

        System.out.println(refund);
    }
}

File: official/docs/golang/v2/referral-customers/refund-by-payment-log.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	client := easypost.New("EASYPOST_API_KEY")

	refund, _ := client.BetaRefundByPaymentLog("paylog_...")

	fmt.Println(refund)
}
